store/file/ods_test.go

```diff
@@ -62,7 +62,9 @@ func TestODSFile(t *testing.T) {
 	t.Cleanup(cancel)
 
 	ODSSize := 8
```

Comment:
It is worth increasing the size so it's bigger than the buffer. Ideally, we would do specify MaxODS to the test suite so it tests everything for all the sizes until the max given. It will take more time, yes, but we have high assurance that all sizes work correctly.